src/commands/voice/volume.ts

```diff
@@ -0,0 +1,41 @@
+import { PermissionLevels } from './../../types/commands.ts';
+import { bot } from "../../../cache.ts";
+import { createCommand, createSubcommand } from "../../utils/helpers.ts";
+
+createCommand({
+  name: "volume",
+  description: "See the current volume.",
+  aliases: ["vol", "v"],
+  arguments: [
+    { name: "subcommand", type: "subcommand", required: false }
+  ],
+  guildOnly: true,
+  execute(message) {
+    const player = bot.lavadenoManager.players.get(message.guildId.toString());
+
+    if (!player) return message.reply("No player in this guild.");
+    
+    return message.reply(`Current volume is ${player?.volume}.`);
+  },
+});
+
+createSubcommand("volume", {
+  name: "set",
+  description: "Set the volume of the player.",
+  aliases: ["s", "change"],
+  permissionLevels: [PermissionLevels.MODERATOR],
+  arguments: [
+    { name: "value", type: "number", minimum: -Infinity, required: true }
```

Comment:
why `-Infinity`? shouldn't this be 0?

Reply:
This may be 0 or -Infinity. Doesn't really matter.

Reply:
Why does it not matter? 0 !== -Infinity. If you check it later in code you could remove it I think.

Reply:
When I tested it, when I use -Infinity then it return the "Please provide..." on - numbers. When I have 0, it doesn't. But this isn't really important, I don't think people will try to set it on -